splash-screen.py

```diff
@@ -0,0 +1 @@
+print("splash-screen")
```

Comment:
tolong ubah isi print jadi seperti ini
```suggestion
print("splash-screen-home")
```